model/Export/QTIPackedItemExporter.php

```diff
@@ -105,10 +105,6 @@ public function exportManifest() {
 		
 		if (!is_null($qtiItem)) {
             
-            //tao currently uses the title as the label of imported qti items:
-            $qtiItem->setAttribute('title', $rdfItem->getLabel());
-            $qtiItemService->saveDataItemToRdfItem($qtiItem, $rdfItem);
-            
```

Comment:
The real source of the issue is that we want to provide a way to match the rdf item metadata to the one in the qti item xml. The real fix should be to definitely define our policy regarding this matching. What is your opinion on that ?